examples/00-EDB/12_edb_sma_connector_on_board.py

```diff
@@ -1,49 +1,50 @@
 """
-EDB: geometry creation
-----------------------
+# EDB: geometry creation
+
 This example shows how to
-1, Create a parameterized PCB layout design.
-2, Place 3D component on PCB.
-3, Create HFSS setup and frequency sweep with a mesh operation.
-4, Create return loss plot
+1. Create a parameterized PCB with an SMA connector footprint for a single-ended
+   SMA connector launch footprint..
+22 Place 3D component on PCB.
+3. Create HFSS setup and frequency sweep with a mesh operation.
+4. Create return loss plot
 """
 ######################################################################
+# ## The Finished Project
 #
-# Final expected project
-# ~~~~~~~~~~~~~~~~~~~~~~
-#
-# .. image:: ../../_static/edb_example_12_sma_connector_on_board.png
-#  :width: 600
-#  :alt: Differential Vias.
-######################################################################
+# <img src="_static\edb_example_12_sma_connector_on_board.png" width="450">
 
 ######################################################################
-# Create parameterized PCB
-# ~~~~~~~~~~~~~~~~~~~~~~~~
-# Initialize an empty EDB layout object on version 2023 R2.
-######################################################################
+# ## Create parameterized PCB
+#
+# Import dependencies.
```

Comment:
We should always use the same wording. Usually it's "Perform required imports." I'm OK with "Import dependencies" but  if you want to use it, always use it. (Note that the inclusion or exclusion of the period also must be consistent.)